bHY6yh-de6884/cod-bqe7s7_bHY6yh-de6884.xml

```diff
@@ -0,0 +1,269 @@
+               <lb ed="#P"/>domum aut fratres / aut sorores / aur
+               <lb ed="#P"/>patrem ut matrem / aut filios / aut agros
+               <lb ed="#P"/>propter nomen meum centuplum accipiet et
+               <lb ed="#P"/>vitam eternam possidebit.</quote>
+            </p>
+            <p xml:id="bHY6yh-de6884-d1e258">
+               OMITTED
+            </p>
+            <p xml:id="bHY6yh-de6884-d1e266">
+               <lb ed="#P"/><ref xml:id="bHY6yh-de6884-Rd1e280">Marchus viii. capitulo.</ref> <quote xml:id="bHY6yh-de6884-Qd1e270">iesus cepit docere
+               <lb ed="#P"/>discipulos suos quoniam oportet filium homnis
+               <lb ed="#P"/>multa pati / et reprobari a senioribus
+               <lb ed="#P"/>et asumis pontificibus et scribis / et
+               <lb ed="#P"/>occidi posttres dies resurgere et palam
+               <lb ed="#P"/>verbum loquebatur et apprehendens eum 
+               <lb ed="#P"/>petrus cepit increpare eum / qui con
+               <lb ed="#P" break="no"/>versus et videns discipulos suos
+               <lb ed="#P"/>comminatus est petro dicens vade
+               <lb ed="#P"/>retro sathanas quia non sapis qui dei
+               <lb ed="#P"/>sunt sed qui hominum sunt.</quote>
+            </p>
+            <p xml:id="bHY6yh-de6884-d1e300"> /
+               <lb ed="#P"/><ref xml:id="bHY6yh-de6884-Rd1e316">Idem x. ca.</ref> <quote xml:id="bHY6yh-de6884-Qd1e302">Quam difficile est confidentes in pe
+               <lb ed="#P" break="no"/>cunias regnum dei intrare.</quote>
+            </p>
+            <p xml:id="bHY6yh-de6884-d1e308">
+               OMITTED
+            </p>
+            <p xml:id="bHY6yh-de6884-d1e321">
+               <lb ed="#P"/><ref xml:id="bHY6yh-de6884-Rd1e341">Lucas xxvi</ref> <quote xml:id="bHY6yh-de6884-Qd1e323">Filii huius seculi prudenti
+               <lb ed="#P" break="no"/>ores st filiis lucis in generatione sua
+               <lb ed="#P"/>ait iesus facite vobis amicos dema
+               <lb ed="#P"/>mona iniquitatis / ut cum defeceritis re
+               <lb ed="#P" break="no"/>cipiant vos in eterna tabernacula.</quote>
+            </p>
+            <p xml:id="bHY6yh-de6884-d1e335">
+               <lb ed="#P"/><ref xml:id="bHY6yh-de6884-Rd1e357">Idem in actibus apostolorum. ca. v.</ref> <quote xml:id="bHY6yh-de6884-Qd1e337">oportet
+               <lb ed="#P"/>obedure deo magis quam hominbus</quote>
+            </p>
+            <p xml:id="bHY6yh-de6884-d1e345">
+               OMITTED
+            </p>
+            <p xml:id="bHY6yh-de6884-d1e361">
+               <lb ed="#P"/><ref xml:id="bHY6yh-de6884-Rd1e388">Jacobus epistula primo. ca. iiii</ref> <quote xml:id="bHY6yh-de6884-Qd1e363">nolite gloriari
+               <lb ed="#P" break="no"/>et men daces esse aduersus veritatem.
+               <pb ed="#P" n="13-v"/>
```

Comment:
@kuriae I'lll merge this in as is, but note that when a new `<pb>` this is also the beginning of a column "a". So everywhere we have a <pb> it should be followed by a <cb n="a"/>

So in this case it should look like: 

```
<pb ed="#P" n="13-v"/>
<cb ed="#P" n="a"/>
<lb ed="#P"/>